CMakeLists.txt

```diff
@@ -531,6 +531,15 @@ else()
   set(CMAKE_CUDA_STANDARD 17)
 endif()
 
+if ((USE_CUDA AND USE_CUTLASS))
+  if(NOT DEFINED CMAKE_CUDA_ARCHITECTURES)
+    set(CMAKE_CUDA_ARCHITECTURES "75;80;86;89;90")
```

Comment:
likely we will have to remove 75 and add 87 in this list

Reply:
A better way is to avoid setting `CMAKE_CUDA_ARCHITECTURES` in cmake file, which might contaminate other downstream projects, but instead configure it in CD:
- Unity: https://github.com/mlc-ai/package/blob/main/scripts/build_mlc_ai_wheel_manylinux.sh#L103
- MLC: https://github.com/mlc-ai/package/blob/main/scripts/build_mlc_chat_wheel_manylinux.sh#L99

Reply:
Also we need to turn on `USE_FLASHINFER` in the CD pipelines

Reply:
I tweaked this PR and moved the logic to `CUDA.cmake` with NVCC version detection. Though it's still part of cmake files. https://github.com/mlc-ai/relax/commit/30c129ef4d43f40c02835a7fa5b485804dbae2c1